src/components/InstitutionBanner/AdminView.tsx

```diff
@@ -21,15 +21,28 @@ const AdminView: FC<AdminViewProps> = ({ banner, type, editMode, setBanner }) =>
       {!editMode ? (
         <>
           <ul>
-            {bannerItems.map(({ icon, item, itemType, defaultText }, key) => (
-              <BannerItem
-                key={key}
-                icon={icon}
-                item={item}
-                itemType={itemType}
-                defaultText={defaultText}
-              />
-            ))}
+            {bannerItems.map(({ icon, item, itemType, defaultText }, key) =>
+              key === bannerItems.length - 1 ? (
+                <>
+                  <br />
+                  <BannerItem
+                    key={key}
+                    icon={icon}
+                    item={item}
+                    itemType={itemType}
+                    defaultText={defaultText}
+                  />
+                </>
+              ) : (
```

Comment:
This conditional might look weird but it's because in the designs the last item in the banner has a gap above it. Here's an example (notice how "Add your charity's address" and "Add your charity's website" have a gap inbetween):

<img width="410" alt="image" src="https://github.com/user-attachments/assets/0f8f83f9-f1ca-4417-b178-6034dcba9e4d">

I figured a quick <br/> would be an easy way to achieve this, but let me know if you have a cleaner solution

Reply:
I would probably go down the route of handling in the css - do li:last-of-type and then add some margin top

Reply:
Thanks for this, that is certainly a much cleaner solution! I actually didn't know about the `last-of-type` css feature- I've learnt something new 👍